frontend/src/metabase/css/components/header.css

```diff
@@ -7,12 +7,6 @@
   color: var(--color-text-dark);
 }
 
-.Header-attribution {
-  display: none; /* disabled */
-  color: var(--color-text-medium);
-  margin-bottom: 0.5em;
-}
```

Comment:
Based on [comment placeholder for frontend/src/metabase/components/Header.jsx], since its usage is removed, this should be removed as well.